src/App.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState } from "react";
+import UserHeader from "./components/header";
+import CharacterCard from "./components/characters";
+import { ContentStyle } from "./styled/style";
+import { characters } from "./data";
+
+type Character = {
+  id?: number;
+  image: string;
+  name: string;
+};
+
+function App() {
+  const [bestScore, setBestScore] = useState(0);
+  const [clicked, setClicked] = useState<number[]>([]);
+  const [score, setScore] = useState(0);
+
+  function shuffle(array: Character[]): void | Character[] {
+    let currentIndex = array.length;
+    let temporaryValue: Character;
```

Comment:
is this being used ?

Reply:
Removed temporary variable, thanks for pointing out the mistake